chainer/functions/activation/softplus.py

```diff
@@ -1,12 +1,13 @@
 import numpy
 
 from chainer import cuda
-from chainer import function
+from chainer import function_node
+from chainer import functions
```

Comment:
Please use `import chainer.functions` .

Reply:
@okuta Thank you for the advice. I fixed it.